src/carplay.cs

```diff
@@ -250,6 +327,49 @@ interface CPInterfaceController {
 
 		[Export ("templates", ArgumentSemantic.Strong)]
 		CPTemplate [] Templates { get; }
+
+		// Not sure about this one. ObjC header says: extern NSString * const CarPlayErrorDomain;
+		[Field ("CarPlayErrorDomain")]
+		NSString CarPlayErrorDomain { get; }
```

Comment:
Is an error domain. grep for it, there are a lot of examples of this.

Reply:
I looked around at some of our existing examples. It appears that ErrorDomain usually goes with an error enum, like this one in CoreText:

```
[Native]
[ErrorDomain ("kCTFontManagerErrorDomain")]
 public enum CTFontManagerError : long {
                None = 0,
...
```

`CarPlayErrorDomain` shouldn't be a field inside of `CPInterfaceController` 👍 

I don't see any enums inside of CarPlay.cs or the CarPlay dir that look like error enums so I am stumped on where this should go.

Reply:
indeed, there's no enum.
```
grep "enum CP" *.g.cs
ios-14.0.g.cs:public enum CPAlertActionStyle : ulong
ios-14.0.g.cs:public enum CPBarButtonStyle : long
ios-14.0.g.cs:public enum CPBarButtonType : ulong
ios-14.0.g.cs:public enum CPTextButtonStyle : long
ios-14.0.g.cs:public enum CPInformationTemplateLayout : long
ios-14.0.g.cs:public enum CPListItemAccessoryType : long
ios-14.0.g.cs:public enum CPListItemPlayingIndicatorLocation : long
ios-14.0.g.cs:public enum CPNavigationAlertDismissalContext : ulong
ios-14.0.g.cs:public enum CPTripPauseReason : ulong
ios-14.0.g.cs:public enum CPPanDirection : long
ios-14.0.g.cs:public enum CPManeuverDisplayStyle : long
ios-14.0.g.cs:public enum CPTimeRemainingColor : ulong
ios-14.0.g.cs:public enum CPTripEstimateStyle : ulong
ios-14.0.g.cs:public enum CPMessageLeadingItem : long
ios-14.0.g.cs:public enum CPMessageTrailingItem : long
ios-14.0.g.cs:public enum CPLimitableUserInterface : ulong
ios-14.0.g.cs:public enum CPContentStyle : ulong
```

 I guess one could be added later... the error domain is rarely used in code (at least for comparison). I suggest you remove it and add this (lack of related enum) in the `.ignore` file. If an enum is added later (beta or release) then it can be added to the _right_ place.